README.md

```diff
@@ -69,7 +92,7 @@ A TypeScript project will have separate `src` and `dist` directories, for source
 |test/|this directory contains your tests|
 |tsconfig.json|the configuration settings for TypeScript compilation|
 
-### TypeScript Configuration
+### <a name="typescriptConfiguration"></a>TypeScript Configuration
 
 [TODO]
 In your `tsconfig.json` file, you can specify options for TypeScript to use when compiling your project. 
```

Comment:
> It's also a good practice to enable strict mode via "strict": true, it can catch many kinds of mistakes at compile time.

I created a follow-up issue to make sure we don't forget about this: https://github.com/strongloop/loopback-next-hello-world/issues/15